src/main/java/com/google/api/codegen/transformer/php/PhpSurfaceNamer.java

```diff
@@ -60,7 +60,7 @@ public void addPageStreamingDescriptorImports(ModelTypeTable typeTable) {
 
   @Override
   public String getClientConfigPath(Interface service) {
-    return "./resources/"
+    return "/resources/"
```

Comment:
I would prefer this to not have the leading slash so that it doesn't imply an absolute path.